canny_baseline/canny_source.c

```diff
@@ -975,7 +975,11 @@ int write_pgm_image(char *outfilename, unsigned char *image, int rows,
    ***************************************************************************/
    if(outfilename == NULL) fp = stdout;
    else{
+#if _WIN32
```

Comment:
@ibogosavljevic, please review Marek's changes in canny_source.c
Do they conflict with your changes in #5?